src/ui/components/panels/loadout/active_loadout/components/_ActionActivationButtons.vue

```diff
@@ -45,6 +45,7 @@ export default Vue.extend({
   name: 'action-activation-buttons',
   props: {
     used: { type: Boolean },
+    fulltech: { type: Boolean, default: false },
```

Comment:
Funnel a `fulltech` option all the way down here so that this common code can be used for Invade.